### PR TITLE
Gateway troubleshooting additions

### PR DESCRIPTION
**Pull request by benolayinka**
#### Summary

- Added sections to close https://github.com/TheThingsIndustries/lorawan-stack-docs/issues/383
- Consolidated all gateway troubleshooting sections and floated important sections to the top of `gateways`.

#### Screenshots

Bildschirmfoto 2021-06-22 um 16 09 09

#### Checklist

- [x] Scope: The referenced issue is addressed, there are no unrelated changes.
- [x] Run Locally: Verified that the docs build using `make server`, posted screenshots, verified external links. Test with `HUGO_PARAMS_SEARCH_ENABLED=true` if style changes will affect the search bar.
- [ ] New Features Marked: Documentation for new features is marked using the `new-in-version` shortcode, according to the guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Style Guidelines: Documentation obeys style guidelines in [CONTRIBUTING](CONTRIBUTING.md).
- [x] Commits: Commit messages follow guidelines in [CONTRIBUTING](CONTRIBUTING.md), there are no fixup commits left.

### CODE REVIEW

File: doc/content/gateways/troubleshooting/_index.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+
+- Link as Gateway to a Gateway Server for traffic exchange, i.e. write uplink and read downlink
+
+**UDP gateways** are unsecured and require no API key.
+
+## How do I see gateway events?
+
+Gateway event logs can be found in the console in the gateway's general information page. See [Working with Events]({{< ref "getting-started/events" >}}) for other ways of subscribing to events.
+
+## How does {{% tts %}} console know whether a gateway is connected?
```
Comment (nejraselimovic) [inline doc/content/gateways/troubleshooting/_index.md:61]:
```suggestion
## How does {{% tts %}} Console know whether a gateway is connected?
```

Comment (nejraselimovic) [reply doc/content/gateways/troubleshooting/_index.md:61]: I think we're using Console with a capital C almost everywhere

**Review by nejraselimovic — APPROVED** [review]

**Review by adriansmares — APPROVED** [review]

File: doc/content/gateways/troubleshooting/_index.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+weight: -2
+---
+
+This section provides help for common issues and frequently asked questions you may have when adding gateways. 
+
+<!--more-->
+
+## What is my server address?
+
+This is the address you use to access {{% tts %}}. If you followed the [Getting Started guide]({{< ref "/getting-started" >}}) this is the same as what you use instead of `thethings.example.com`. If you use {{% tts %}} Cloud, this is typically `tenant_id.region.cloud.thethings.industries`.
```
Comment (neoaggelos) [inline doc/content/gateways/troubleshooting/_index.md:14]: Good to include the URL for The Things Stack Community Edition as well.

**Review by neoaggelos — COMMENTED** [review]
LGTM in general, left a comment.

**Pull request merged**